cpp/ps0007/ps0007_sol.cbp

```diff
@@ -29,10 +29,13 @@ double thearea(int shape, double dimensions[]) {
 }
 
 int main() {
+    // prompting user for the shape 1.5/1.5
     int shape;
     double dimensions[2];
     cout << "Enter shape code (1 - Rectangle, 2 - Square, 3 - Circle): ";
     cin >> shape;
```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input